src/components/Datepicker.tsx

```diff
@@ -68,7 +70,9 @@ const Datepicker: React.FC<Props> = ({
     readOnly = false,
     minDate = null,
     maxDate = null,
-    disabledDates = null
+    disabledDates = null,
+    inputId,
```

Comment:
I think we should consider having the `id` and `name` be optional

Reply:
They are optional in the type. They just don't accept null. id and name do not accept null as a value on an input field so this better represents the actual type where the values are used.